### PR TITLE
Add `Fukuoka Ruby Award 2019`.

### PR DESCRIPTION
**Pull request by digitalfukuoka**
Please review and merge.

### CODE REVIEW

File: en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: "2019 Fukuoka Ruby Award Competition - Entries to be judged by Matz"
+author: "Fukuoka Ruby"
+translator:
+date: 2018-11-29 00:00:00 +0000
+lang: en
+---
+
+Dear Ruby Enthusiasts,
+
+The Government of Fukuoka, Japan together with "Matz" Matsumoto would like to invite you to enter the following Ruby competition. If you have developed an interesting Ruby program, please be encouraged to apply.
```
Comment (JuanitoFatas) [inline en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
The Government of Fukuoka, Japan, together with "Matz" Matsumoto would like to invite you to enter the following Ruby competition. If you have developed an interesting Ruby program, please be encouraged to apply.
```

File: en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+This year, there are the following special prizes:
+
+The winner of the AWS Prize will receive:
+
+* Amazon Echo (subject to change)
+* AWS architect technical consultation
+
+The winner of the GMO Pepabo Prize will receive:
+
+* Lolipop! shared hosting service: 10-year free subscription of the Standard Plan,or JPY 100,000 coupon for the metered plan of the Managed Cloud
```
Comment (JuanitoFatas) [inline en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
* Lolipop! shared hosting service: 10-year free subscription of the Standard Plan, or JPY 100,000 coupon for the metered plan of the Managed Cloud
```

File: en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Muumuu Domain DNS registration service: 10-year free subscription of one domain(for a domain that costs JPY 10,000 or less per year)
```
Comment (JuanitoFatas) [inline en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
* Muumuu Domain DNS registration service: 10-year free subscription of one domain (for a domain that costs JPY 10,000 or less per year)
```

File: en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+The winner of the IIJ GIO Prize will receive:
+
+* IIJ GIO free coupon worth 500,000 yen(Up to 6 months)
```
Comment (JuanitoFatas) [inline en/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md]:
```suggestion
* IIJ GIO free coupon worth 500,000 yen (Up to 6 months)
```

**Comment by digitalfukuoka** [comment]
Tnank you very much for correction.

**Comment by JuanitoFatas** [comment]
Applied corrections and merged as 8f90080.

If anything is wrong, please submit a Pull Request to fix it, thank you!

**Pull request closed without merging**